election/client/nodejs/Dockerfile

```diff
@@ -1,3 +1,5 @@
-FROM node:0.12
+FROM node:6.3-slim
```

Comment:
Major bump :). Is this basically latest? (disclosure: I know nothing about node or javascript).